project/Dependencies.scala

```diff
@@ -420,7 +420,7 @@ object Dependencies {
   ) ++ slf4jBindingDependencies // During testing, add an slf4j binding for _all_ libraries.
 
   val kindProjectorPlugin = "org.spire-math" %% "kind-projector" % kindProjectorV
-  val paradisePlugin = "org.scalamacros" % "paradise" % paradiseV
+  val paradisePlugin = "org.scalamacros" % "paradise" % paradiseV cross CrossVersion.full
```

Comment:
I'm surprised this is necessary if we're not doing the cross-build.

Reply:
(Yesterday) i Learned:


- `CrossVersion.binary` (the default for scala libs) means "this library has specific builds for `2.11`, `2.12`, `2.13`, etc." Libraries using this default can use `%%` as the [artifact separator](https://www.scala-sbt.org/1.0/docs/Cross-Build.html#Cross-Building+a+Project).
- `CrossVersion.full` means "the library has specific builds for `2.12.0`, `2.12.1`, `2.12.2`, etc." Checkout the ton of cross versions for https://mvnrepository.com/artifact/org.scalamacros/paradise